cpp/installer.cc

```diff
@@ -75,7 +76,7 @@
 
 
 [[noreturn]] void Usage() {
-    ::Usage(std::string("--ub-tools-only|--fulltext-backend|(vufind_system_type [--omit-cronjobs] [--omit-systemctl])\n") +
+    ::Usage(std::string("(--production|--test) --ub-tools-only|--fulltext-backend|(vufind_system_type [--omit-cronjobs] [--omit-systemctl])\n") +
```

Comment:
Does the usage still make sense like this?
- Lacking documentation => what does production / test acually mean in which system context?
- Is it still necessary to have --omit-cronjobs and --omit-systemctl flags or is this indirectly defined via the production flag?

Reply:
Well, production means production.  That is, if there is a difference between a test environment and a production environment this would select for that.  There are, possibly several, installation types where this has no effect so far as they have no differences or concepts of what a production or test environment are.  An example is probably if you choose --ub-tools-only.  Then either flag is fine.

Reply:
On second thought we definitely need those as they are needed for the difference between sobek and ptah on one side and the Docker containers on the other side.